web/impact/impact/tests/test_organization_list_view.py

```diff
@@ -27,11 +32,45 @@ def test_get_startup(self):
 
     def test_get_partner(self):
         count = 5
-        partners = PartnerFactory.create_batch(5)
+        partners = PartnerFactory.create_batch(count)
         with self.login(username=self.basic_user().username):
             url = reverse("organization")
             response = self.client.get(url)
             assert response.data['count'] == count
             assert all([serialize_org(partner.organization)
                         in response.data['results']
                         for partner in partners])
+
+    def test_updated_at_lt_datetime_filter(self):
+        count = 5
+        organizations = OrganizationFactory.create_batch(count)
+        response = ""
+        lastweek = datetime.datetime.now(pytz.utc) - datetime.timedelta(days=7)
+        Organization.objects.filter(id__in=[
+            organizations[0].id,
+            organizations[1].id]).update(
+            updated_at=lastweek)
+        with self.login(username=self.basic_user().username):
+            url = "{base_url}?updated_at__lt={datestr}".format(
+                base_url=reverse("organization"),
+                datestr=lastweek.strftime("%Y-%m-%dT%H:%M:%S.%fZ"))
+            response = self.client.get(url)
+        json_response = json.loads(response.content)
+        self.assertEqual(json_response['count'], 0)
+
+    def test_updated_at_gt_datetime_filter(self):
+        count = 5
+        organizations = OrganizationFactory.create_batch(count)
+        response = ""
+        lastweek = datetime.datetime.now(pytz.utc) - datetime.timedelta(days=7)
+        Organization.objects.filter(id__in=[
+            organizations[0].id,
+            organizations[1].id]).update(
+            updated_at=lastweek)
+        with self.login(username=self.basic_user().username):
+            url = "{base_url}?updated_at__gt={datestr}".format(
+                base_url=reverse("organization"),
+                datestr=lastweek.strftime("%Y-%m-%dT%H:%M:%S.%fZ"))
+            response = self.client.get(url)
+        json_response = json.loads(response.content)
+        self.assertTrue(json_response['count'] > 0)
```

Comment:
I'd test here that the result you want is actually there. Otherwise it might be vulnerable to cross-test leakage.